zerver/lib/actions.py

```diff
@@ -1704,6 +1707,22 @@ def check_send_private_message(sender: UserProfile, client: Client,
 
     return do_send_messages([message])[0]
 
+@has_request_variables
+def check_send_webhook_message(request: HttpRequest, user_profile: UserProfile,
+                               topic: Text, body: Text,
+                               stream: Optional[Text]=REQ(default=None),
+                               custom_topic: Optional[Text]=REQ("topic", default=None)) -> None:
+    if stream is None:
+        assert user_profile.bot_owner is not None
+        check_send_private_message(user_profile, request.client,
+                                   user_profile.bot_owner, body)
+    else:
+        subject = topic
+        if custom_topic is not None:
+            subject = custom_topic
```

Comment:
If you're going to use a new variable name, I'd do it in an `if/else` format.  But generally they're synonyms, and we're hoping to slowly erase "subject" from the codebase.